.github/workflows/Build-Test.yml

```diff
@@ -0,0 +1,90 @@
+
+# Run this workflow every time a new commit pushed to your repository
+on:
+  push:
+
+jobs:
+  tests:
+    if: "! contains(toJSON(github.event.commits.*.msg), 'skip') && ! contains(toJSON(github.event.commits.*.msg), 'ci')" #skip ci...
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ubuntu-16.04,ubuntu-18.04,ubuntu-20.04]
```

Comment:
sorry to only notice now, but why build on all ubuntu LTS versions? on travis-ci we only built with whatever the default environment was. i would suggest that ubuntu-20.04 is enough. what we do care about is the various php minor versions, but we don't expect changes between ubuntu versions. that would reduce the number of builds from 13 to 5 if i understand correctly.

Reply:
fixed with latest commit.